dask/dataframe/accessor.py

```diff
@@ -129,6 +129,7 @@ def split(self, pat=None, n=-1, expand=False):
                 delimiter = " " if pat is None else pat
                 meta = type(self._series._meta)([delimiter.join(["a"] * (n + 1))])
```

Comment:
Small suggestion, to make this more consistent with the rest of dask.

```suggestion
                meta = self._series._constructor([delimiter.join(["a"] * (n + 1))], index=self._series._meta_nonempty[:1].index)
```

That may not be properly linted.

Reply:
The `self._series._meta_nonempty[:1].index` is certainly nice. But if I'm not mistaken, the `self._series._constructor` needs to be called with `dsk, name, meta, divisions` (cf. `new_dd_object(dsk, name, meta, divisions)`). Hence, your suggestion results in `TypeError: new_dd_object() got an unexpected keyword argument 'index'`.

Maybe I miss the point.

Reply:
Whoops, sorry. Should be `self._series._meta._constructor`. That's similar to `type(self._series)(...)`, but a bit more correct (newer parts of dask are using the `._constructor` approach).